Assets/Scripts/Models/Furniture.cs

```diff
@@ -143,15 +146,25 @@ private Furniture(Furniture other)
         UnlocalizedDescription = other.UnlocalizedDescription;
     }
 
+    /// <summary>
+    /// This event will trigger when the furniture has been changed.
```

Comment:
I agree with that.

We could also make more events that replace this one to be more specific, or just leave it like that.
Maybe as a separate issue/pr